release-notes/release-note-template.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: ballerina-blank-page
+title: Release Note
+---
+### Overview of Ballerina Swan Lake <VERSION>
+
+This <VERSION> release includes the language features planned for the Ballerina Swan Lake release. Moreover, this release includes improvements and bug fixes to the language, runtime, standard library, code to cloud, and developer tooling. This release note lists only the features and updates added after the <VERSION> release of Ballerina Swan Lake.
```

Comment:
What if we use `The x.x.x release ...`?

Reply:
@nadeeshaan - Sorry, this isn't clear. How can we use `The x.x.x release ...` for Swan Lake and where should it be changed?

Reply:
@praneesha I meant, instead of `This <version> release` part what if we use  `The <Version> release`

Reply:
@MaryamZi - WDYT?

Reply:
Shouldn't it be

```suggestion
This release includes the language features planned for the Ballerina Swan Lake release. Moreover, this release includes improvements and bug fixes to the language, runtime, standard library, code to cloud, and developer tooling. This release note lists only the features and updates added after the <VERSION> release of Ballerina Swan Lake.
```

or

```suggestion
The Swan Lake Alpha3 release includes the language features planned for the Ballerina Swan Lake release. Moreover, this release includes improvements and bug fixes to the language, runtime, standard library, code to cloud, and developer tooling. This release note lists only the features and updates added after the <VERSION> release of Ballerina Swan Lake.
```

I don't think we have to mention the version if we use "This".

Reply:
That was used as an alias since this is the template. We will need to change it in each release once this template is copied.

Reply:
Basically 

```suggestion
This release includes the language features planned for the Ballerina Swan Lake release. Moreover, this release includes improvements and bug fixes to the language, runtime, standard library, code to cloud, and developer tooling. This release note lists only the features and updates added after the <VERSION> release of Ballerina Swan Lake.
```

or

```suggestion
The <VERSION> release includes the language features planned for the Ballerina Swan Lake release. Moreover, this release includes improvements and bug fixes to the language, runtime, standard library, code to cloud, and developer tooling. This release note lists only the features and updates added after the <VERSION> release of Ballerina Swan Lake.
```